src/assets/wise5/components/table/tableController.ts

```diff
@@ -1123,6 +1204,14 @@ class TableController extends ComponentController {
     }
   }
 
+  updateDataExplorerSeriesNames() {
+    for (const singleDataExplorerSeries of this.dataExplorerSeries) {
```

Comment:
Nit: I think we should just use series here instead of singleDataExplorerSeries. It makes the code shorter and easier to read. What do you think?